src/profile/v100/address/non_fungible_resource_address.rs

```diff
@@ -177,6 +177,18 @@ impl HasSampleValues for NonFungibleResourceAddress {
 }
 
 impl NonFungibleResourceAddress {
+    pub fn random(network_id: NetworkID) -> Self {
```

Comment:
Uh... this is copy paste :/ but kind of tricky to shoehorn support for NonFungibleResourceAddr in the macro. Perhaps ok for now